.github/workflows/e2e-windows.yaml

```diff
@@ -0,0 +1,159 @@
+
+      - name: 🚧 Setup Terraform
+        uses: hashicorp/setup-terraform@b9cd54a3c349d3f38e8881555d616ced269862dd
+        with:
+          terraform_wrapper: false
+          terraform_version: "1.10.4"
+
+      - name: Install Azure CLI
+        shell: powershell
+        run: |
+          # Download and install Azure CLI for Windows
+          $azCliUrl = "https://aka.ms/installazurecliwindows"
+          $azCliPath = "$env:TEMP\AzureCLI.msi"
+          
+          # Download Azure CLI MSI
+          Invoke-WebRequest -Uri $azCliUrl -OutFile $azCliPath
+          
+          # Install Azure CLI
+          Start-Process msiexec.exe -Wait -ArgumentList "/I $azCliPath /quiet"
+          
+          # Add Azure CLI to PATH for this session
+          $env:PATH += ";C:\Program Files (x86)\Microsoft SDKs\Azure\CLI2\wbin"
+          echo "C:\Program Files (x86)\Microsoft SDKs\Azure\CLI2\wbin" | Out-File -FilePath $env:GITHUB_PATH -Encoding utf8 -Append
+          
+          # Verify installation
+          az --version
+
+      - name: 'Az CLI login - federated'
+        uses: azure/login@v2
+        with:
+          client-id: ${{ secrets.AZURE_OID_CLIENT_ID }}
+          tenant-id: ${{ secrets.MPF_TENANTID }}
+          subscription-id: ${{ secrets.MPF_SUBSCRIPTIONID }}
+
+      - name: 🧪 Run ARM CLI Tests
+        shell: powershell
+        run: |
+          $output = go run ./cmd arm --templateFilePath ./samples/templates/aks-private-subnet.json --parametersFilePath ./samples/templates/aks-private-subnet-parameters.json
+          $lineCount = ($output | Measure-Object -Line).Lines
+          Write-Host "Result line count: $lineCount"
+          if ($lineCount -ne 13) {
+            Write-Host "Expected 13 lines, got $lineCount"
+            exit 1
+          }
+          Write-Host "Test passed: $lineCount lines"
+
+      - name: 🧪 Run Bicep CLI Tests
+        shell: powershell
+        run: |
+          $output = go run ./cmd bicep --bicepFilePath ./samples/bicep/aks-private-subnet.bicep --parametersFilePath ./samples/bicep/aks-private-subnet-params.json
+          $lineCount = ($output | Measure-Object -Line).Lines
+          Write-Host "Result line count: $lineCount"
+          if ($lineCount -ne 13) {
+            Write-Host "Expected 13 lines, got $lineCount"
+            exit 1
+          }
+          Write-Host "Test passed: $lineCount lines"
+
+      - name: 🧪 Run Terraform CLI Tests
+        shell: powershell
+        run: |
+          # Use Get-Command to dynamically find Terraform path (PowerShell best practice)
+          $env:MPF_TFPATH = (Get-Command terraform).Source
+          Write-Host "Terraform path: $env:MPF_TFPATH"
+          
+          $output = go run ./cmd terraform --workingDir ./samples/terraform/aci --varFilePath ./samples/terraform/aci/dev.vars.tfvars
+          $lineCount = ($output | Measure-Object -Line).Lines
+          Write-Host "Result line count: $lineCount"
+          if ($lineCount -ne 13) {
```

Comment:
this should be 12 due to difference in powershell and bash script handling newlines